script.js

```diff
@@ -243,3 +243,47 @@ ${dynamicImage}
     }
   });
 }
+
+// Javascript for the local storage
+const userDataInLocalStorage = localStorage.getItem('userData');
+const parsedData = JSON.parse(userDataInLocalStorage);
+document.getElementById('fullName').value = parsedData.name;
+document.getElementById('email').value = parsedData.email;
+document.getElementById('textArea').value = parsedData.comment;
+function localStorageMine() {
+  const userName = document.getElementById('fullName');
+  const userEmail = document.getElementById('email');
+  const userComment = document.getElementById('textArea');
+  const uname = userName.value;
+  const email = userEmail.value;
+  const comment = userComment.value;
+  const inputFieldObject = {
+    name: uname,
+    email,
+    comment,
+  };
+  try {
+    const forLocalStorage = JSON.stringify(inputFieldObject);
+    window.localStorage.setItem('userData', forLocalStorage);
+  } catch (e) {
+    console.log('error occured while storing data to localStorage');
+  }
+}
+
+// console.log(localStorage.getItem('userData'));
```

Comment:
To follow our [list of best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md), please do not commit old pieces of code as inline comments. They will make your project look messy. If you need to review a previous version of your code, you can always use git history. 🤗